lib/ws/style/recursive_open_struct_use.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Ws
+  module Style
+    # This cop flags uses of RecursiveOpenStruct. RecursiveOpenStruct is a library used in the
+    # Wealthsimple ecosystem that is being phased out due to security issues.
+    #
+    # RecrusiveOpenStruct inherits from OpenStruct, which is now officially discouraged to be used
+    # for performance, version compatibility, and security issues.
```

Comment:
**[Suggestion]**
Consider adding the link here: https://ruby-doc.org/stdlib-3.0.1/libdoc/ostruct/rdoc/OpenStruct.html#class-OpenStruct-label-Caveats